[2024-11-18T22:40:04.858+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T22:40:04.881+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T22:30:00+00:00 [queued]>
[2024-11-18T22:40:04.893+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T22:30:00+00:00 [queued]>
[2024-11-18T22:40:04.894+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T22:40:04.915+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_api_sales_data> on 2024-11-18 22:30:00+00:00
[2024-11-18T22:40:04.922+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=134) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T22:40:04.924+0000] {standard_task_runner.py:72} INFO - Started process 136 to run task
[2024-11-18T22:40:04.925+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'fetch_api_sales_data', 'scheduled__2024-11-18T22:30:00+00:00', '--job-id', '515', '--raw', '--subdir', 'DAGS_FOLDER/raw/__pycache__/fetch_data.py', '--cfg-path', '/tmp/tmp6x5oo0vh']
[2024-11-18T22:40:04.927+0000] {standard_task_runner.py:105} INFO - Job 515: Subtask fetch_api_sales_data
[2024-11-18T22:40:04.988+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T22:30:00+00:00 [running]> on host c06adc74012b
[2024-11-18T22:40:05.104+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='fetch_api_sales_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T22:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T22:30:00+00:00'
[2024-11-18T22:40:05.106+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T22:40:05.107+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T22:40:05.107+0000] {logging_mixin.py:190} INFO - Current task name:fetch_api_sales_data state:running start_date:2024-11-18 22:40:04.882832+00:00
[2024-11-18T22:40:05.108+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T22:40:05.109+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T22:40:06.415+0000] {fetch_data.py:29} INFO - Successfully fetched sales data from the API.
[2024-11-18T22:40:06.442+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T22:40:06.470+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T22:40:06.496+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=fetch_api_sales_data, run_id=scheduled__2024-11-18T22:30:00+00:00, execution_date=20241118T223000, start_date=20241118T224004, end_date=20241118T224006
[2024-11-18T22:40:06.519+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T22:40:06.526+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T22:40:06.528+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 22:40:00.410638+00:00
[2024-11-18T22:40:06.529+0000] {logging_mixin.py:190} INFO - Task hostname:c06adc74012b operator:PythonOperator
[2024-11-18T22:40:06.544+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T22:40:06.576+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T22:40:06.580+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
